apps/web-client/src/modules/details/components/episodes-section.tsx

```
@@ -213,81 +226,108 @@ export function EpisodesSection({
       progressData.seasons.forEach((s) => {
         prevMap.set(s.seasonNumber, [...s.watchedEpisodeIds]);
       });
       previousWatchedIdsRef.current = prevMap;
     }
-    selectedEpisodesRef.current = selectedMap;
+    selectedEpisodesRef.current = toMark;
+
+    try {
+      if (toUnmark.size > 0) {
+        const idsToUnmark: string[] = [];
+        toUnmark.forEach((ids) => idsToUnmark.push(...ids));
```

Biome (2.4.7)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)